pytest.ini

```diff
@@ -1,2 +1,3 @@
 [pytest]
-addopts = --ignore docs/apply_geometry.ipynb --ignore docs/xpd_examples.ipynb --ignore "docs/agipd_geometry.ipynb" --ignore docs/xgm_two_runs.ipynb
+addopts = --ignore docs/apply_geometry.ipynb --ignore docs/xpd_examples.ipynb --ignore "docs/agipd_geometry.ipynb" --ignore docs/xgm _two_runs.ipynb
```

Comment:
You've accidentally introduced a space in `xgm_two_runs`

Reply:
oops, indeed thanks for correcting that

Reply:
Well, I did now